### DISCUSSION TITLE
Tracker returned error: Your client is not on the whitelist

### DISCUSSION THREAD

**Discussion by xiataojiaxuan** [transmission/transmission#8473] [Support]
### What is the issue?

Guys what means is that, how can I fix it?
<img width="511" alt="Screenshot 2024-09-21 at 11 18 13 AM" src="https://github.com/user-attachments/assets/c83d0407-9713-4a8f-91c2-8dcb52cd3bd8">

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.0.6

**Comment by Pentaphon** [comment]
Your tracker banned 4.0.6 due to a bug. Downgrade to 4.0.5 for now.

**Comment by xiataojiaxuan** [comment]
> Your tracker banned 4.0.6 due to a bug. Downgrade to 4.0.5 for now.

it worked, thanks!
How can I add it to the whitelist btw?

**Comment by Pentaphon** [comment]
> How can I add it to the whitelist btw?

Trackers won't add it because of that bug so you will have to wait until 4.0.7 or higher to upgrade again. Check your tracker's forum for more info.

**Comment by robd003** [comment]
@Pentaphon Would you happen to know what's going on with the 4.0.x branch? It looks like it's been neglected for 4 months now.

**Comment by Pentaphon** [comment]
> Would you happen to know what's going on with the 4.0.x branch? It looks like it's been neglected for 4 months now.

No idea but we should keep discussion of this at https://github.com/transmission/transmission/issues/7094

**Comment by dbrisinda** [comment]
I upgraded to 4.1.0-beta.2 and it still returns this error for some torrents.

**Comment by ckerr** [comment]
> I upgraded to 4.1.0-beta.2 and it still returns this error for some torrents.

Probably because your tracker doesn't whitelist beta releases?
